### PR TITLE
Phys prop mag susc

### PR DESCRIPTION
**Pull request by dccowan**
Created physical properties-magnetic susceptibility section.

### CODE REVIEW

**Comment by lheagy** [comment]
It looks like it is failing because the file "magnetic_permeability_mathematical_relationships" is not in any of the table of contents. Could you check the index file inside of the magnetic permeability folder to make sure it is included (and spelled correctly)??

build 729 - ubcgif em - travis ci 2016-02-16 19-31-08

File: content/physical_properties/magnetic_permeability/index.rst [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
 **Contents:**
 
  .. toctree::
     :maxdepth: 2
 
-    magnetic_permeability_lab_setup_measurements
+    magnetic_permeability_definition
     magnetic_permeability_units_values
     magnetic_permeability_factors
-    magnetic_permeability_mathematical_relationships
```
Comment (lheagy) [inline content/physical_properties/magnetic_permeability/index.rst]: this deletion is causing the fail

**Comment by lheagy** [comment]
@dccowan : these merge conflicts need to be resolved (ie locally merge master, resolve the conflicts there and then push those changes) before we can merge this branch in. I am happy to sit with you on monday if you would like a hand with this

**Comment by dccowan** [comment]
Sure. I can do that when I get in this morning.

**Comment by lheagy** [comment]
looks like this one is good to go :)

**Pull request merged**